### ISSUE TITLE
can not open the menu

### ISSUE THREAD

**Issue by RageStone** [400plus/400plus#456]
installed 400plus on the camera. when I boot up the camera the blue led lights up briefly like it says it should,  but when I click it it doesnt open any menu. it could be the fact that im on firmware 1.0.4 but I just couldnt find an installation for 1.1.1
I know this project is abandoned and old, but please help someone!

**Comment by nicolasvila** [comment]
There are many places where you can find the firmware. You can find version 1.1.1 for EOS 400D on this archived snapshot of the Canon website:
https://web.archive.org/web/20080507103407/http://web.canon.jp/imaging/eosdigital3/e4kr3_firmware-e.html
I checked for you the integrity of the file.
md5sum: e3dafd73b8ac89b9dddc83d757363742
sha256sum: 9a2c48a9a6989a6138ccee38614d1977ffe2db584db1fd84984cd573c738346b

Some time ago I also lost the menu, while using 1.1.1 firmware. If you face the same case, simply reinstall the 400plus firmware file on the card, which may be corrupted. I faced the same issue and I overwrote the firmware and language file on the memory card.
https://github.com/400plus/400plus/releases/tag/20160404-10

**Comment by eduperez** [comment]
It is absolutely necessary to update the camera to version 1.1.1 of the firmware; it is dangerous to try to use 400plus on a camera with a different version of the firmware.

The firmware update file, even if it was distributed for free, is proprietary of Canon, and we cannot redistribute it publicly; however, I can send it to you privately, if you share an email address.

**Comment by nicolasvila** [comment]
@eduperez you're right. I changed the link in my comment so that the firmware can be downloaded directly from web.archive.org ;-)
